ex3.html

```diff
@@ -0,0 +1,558 @@
+									</select>
+									</li>
+									<li>Access modifiers
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>Design Patterns
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>Issue Tracking
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>Polymorphism
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>RegEx
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+								</ul>
+
+							</fieldset>
+
+
+
+							<fieldset>
+								<legend>HTTP</legend>
+
+								<ul>
+									<li>Protocol
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Response codes
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>REST
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Headers
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Request methods
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Sessions & Cookies
+										<select>
```

Comment:
I'm just going to call this out once more here and say that we should probably address these unescaped HTML entities throughout.